src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -145,20 +150,98 @@ private void create(RoutingContext routingContext) {
         response.setParentInstances(newInstance.getParentInstances());
         response.setChildInstances(newInstance.getChildInstances());
         updateInstanceRelationships(response, routingContext, context,
-                x -> {
-                    try {
-                      URL url = context.absoluteUrl(String.format("%s/%s",
-                        INSTANCES_PATH, success.getResult().getId()));
-                      RedirectResponse.created(routingContext.response(), url.toString());
-                    } catch (MalformedURLException e) {
-                      log.warn(
-                        String.format("Failed to create self link for instance: %s", e.toString()));
-                    }
-                }
+          x -> {
+            try {
+              URL url = context.absoluteUrl(String.format("%s/%s",
+                INSTANCES_PATH, success.getResult().getId()));
+              RedirectResponse.created(routingContext.response(), url.toString());
+            } catch (MalformedURLException e) {
+              log.warn(
+                String.format("Failed to create self link for instance: %s", e.toString()));
+            }
+          }
         );
       }, FailureResponseConsumer.serverError(routingContext.response()));
   }
 
+  /**
+   * Creates a collection of Instances expecting the request could be mapped to Instances schema.
+   *
+   * @param routingContext context for the handling of a request in Vert.x-Web
+   */
+  private void createCollection(RoutingContext routingContext) {
+    WebContext webContext = new WebContext(routingContext);
+    JsonObject request = routingContext.getBodyAsJson();
+    JsonArray instanceCollection = request.getJsonArray("instances", new JsonArray());
+    Integer totalRecordsRq = request.getInteger("totalRecords", 0);
+    log.info("Received collection of Instances with size:" + totalRecordsRq);
```

Comment:
This reports the number of records the client says is present, not how many are actually in the array